src/channels/views/ChannelsList/ChannelsList.tsx

```diff
@@ -14,23 +18,52 @@ interface ChannelsListProps {
   params: ChannelsListUrlQueryParams;
 }
 
-export const ChannelsList: React.FC<ChannelsListProps> = () => {
+export const ChannelsList: React.FC<ChannelsListProps> = ({ params }) => {
   const navigate = useNavigator();
   const { data, loading } = useChannelsList({ displayLoader: true });
 
+  const channelsChoices =
+    params.id &&
+    data?.channels
+      ?.map(channel => ({
+        label: channel.name,
+        value: channel.id
+      }))
+      .filter(channel => channel.value !== params.id);
+
   const navigateToChannelCreate = () => navigate(channelAddUrl);
 
-  const onRemove = () => null;
+  const handleRemoveConfirm = () => null;
```

Comment:
It won't remove the channel, but the referenced task explains it should -  [ https://app.clickup.com/t/6a5vav](https://app.clickup.com/t/6a5vav)

Reply:
I updated task description